apps/src/gamelab/GameLab.js

```diff
@@ -779,6 +780,12 @@ GameLab.prototype.runButtonClick = function () {
   this.studioApp_.attempts++;
   this.execute();
 
+  //Log song count in Dance Lab
+  if (this.isDanceLab && experiments.isEnabled("songSelector")) {
+    const song = getStore().getState().songSelector;
```

Comment:
I believe this should now be `selectedSong` instead of `songSelector`.

the time delay between making logging changes and discovering they're not working can be a pain. unless there's an easy way to see what's getting logged to GA in development mode, it might be worth breaking in here to make sure the right value is getting passed in.

Reply:
Thanks for catching that - I was able to get into GA and check that this is working as expected.